eslint.config.mjs

```diff
@@ -2,7 +2,9 @@
 
 import eslint from '@eslint/js'
 import prettierRecommended from 'eslint-plugin-prettier/recommended'
+// eslint-disable-next-line import/no-unresolved
```

Comment:
I think this is due to https://github.com/import-js/eslint-plugin-import/issues/1485 which includes a fix that works for me (locally) but produces a couple extra warnings.